IMPLEMENTATION_GUIDE.md

```diff
@@ -1373,17 +1427,54 @@ Create an Amazon Translate Parallel Data. If the parallel_data already exists, o
 
 Body:
 
+* This is a proxy for boto3 create_parallel_data function. The JSON dictionary provided in the POST payload must reflect the named parameters specified in the [boto3 documentation](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/transcribe.html#TranslateService.Client.create_parallel_data).
+
 ```
 {
-    'parallel_data_name'='string',
-    'parallel_data_csv='string'
+  "Name"="string",
```

Comment:
I've changed the function signature for create_parallel_data because we need to allow people to use the encryption options. Now, our proxy function is identical to the boto3 function signature. Even though this technically breaks backwards compatbilty, I don't think anyone was using the parallel_data functions in the previous MIE release (version 3.0.4)

Reply:
The MIE version is hardcoded in https://github.com/aws-solutions/aws-media-insights-content-localization/blob/e94b109f455ef516406022763caa8024853cadeb/deployment/content-localization-on-aws.yaml#L27. We can refer to that line to find the MIE / CL compatibility mapping.